components/event-service/server/server.go

```diff
@@ -63,6 +64,7 @@ func (s *Server) initRegistry() map[string][]string {
 	registry[NodeTerminated] = []string{config.CFG_KEY}
 	registry[ProjectRulesUpdate] = []string{config.CFG_KEY, config.COMPLIANCE_INGEST_KEY}
 	registry[ProjectRulesUpdateStatus] = []string{config.AUTHZ}
+	registry[ProjectRulesCancelUpdate] = []string{config.CFG_KEY, config.COMPLIANCE_INGEST_KEY}
```

Comment:
Registering ingest-service and compliance-service for project rules cancel update events.